api/lib/inet-grouping.js

```diff
@@ -103,3 +103,42 @@ export const mapRequestToSubnet = (req) => {
 
   return addr
 }
+
+/**
+ * @param {import('pg').Client} pgClient
+ * @param {import('node:http').IncomingMessage} req
+ * @param {string} stationId
+ * @param {string} inetGroup
+ * @param {function} recordTelemetryFn
+ */
+export const logNetworkInfo = async (pgClient, headers, stationId, inetGroup, recordTelemetryFn) => {
```

Comment:
I agree it's better to keep the name `inet-grouping.js`. However, I don't think this new function `logNetworkInfo` belongs to a file with such a name.

Can we put this function to a new file please? For example `network-info-logger.js`.

@juliangruber what's your opinion?

Reply:
Since it's only used by one file, I'd also like to inline it in that file. Otherwise your proposal works for me too

Reply:
Addressed in https://github.com/filecoin-station/spark-api/pull/339/commits/66b9d3e08937267824c36c61f708c182230d187e. I did 'network-info-logger.js' to avoid a weird const `stationIdsSeen` at the top of index.js, and for a good place to put unit tests.

Let me know if either of you prefer it in index.js

Reply:
I think this is good enough for now 👍🏻